apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -808,7 +809,7 @@ class ManageStudentsTable extends Component {
           {isSectionAssignedCSA &&
             experiments.isEnabled(experiments.CODE_REVIEW_GROUPS) && (
               <ManageCodeReviewGroups
-                sectionId={sectionId}
+                dataApi={new CodeReviewGroupsDataApi(sectionId)}
```

Comment:
Is the api object created in here for testing purposes? just asking since it does seem a little weird that the ManageStudentsTable has awareness of the data api which seems like it should be internal to code review groups components, but I can also understand if that's needed for testing

Reply:
yeah just for testing purposes, so we can stub out the api. I can add a comment that mentions that?

Reply:
Added a comment